royal_panda_manipulation_controller:
  type: mppi_manipulation/RoyalPandaControllerRos
  fixed_base: $(arg fixed_base)
  linear_weight: 10.0
  angular_weight: 1.0
  obstacle_radius: 0.1

  # TODO(giuseppe) set to the right ones
  grasp_translation_offset: [ 0.0, 0.0, 0.0 ]
  grasp_orientation_offset: [ 0.0, 0.0, 0.0, 1.0 ]

  object_reference_position: 1.50   # a bit less than pi/2

  arm_id: panda
  joint_names: [ "panda_joint1", "panda_joint2", "panda_joint3", "panda_joint4", "panda_joint5", "panda_joint6", "panda_joint7" ]
  k_gains: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  d_gains: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0]

  publish_rate: 10.0
  coriolis_factor: 1.0